common/sources.gni

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2020 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+brave_chrome_common_sources = [
+  "//brave/common/brave_content_client.cc",
+  "//brave/common/brave_content_client.h",
+  "//brave/common/brave_channel_info.cc",
```

Comment:
can you please sort the list? e.g. with `gn format`

Reply:
Done.